app/assets/stylesheets/components/_offer_card.scss

```diff
@@ -0,0 +1,40 @@
+.cards-container {
```

Comment:
This should not be here! I deleted it before pushing, but apparently it didn't take.

Anyhow, it won't affect anything because such a class no longer exists. Will be overwritten with next push.